src/api/views.py

```diff
@@ -20,6 +20,7 @@
 from api.serializers import ValidateSerializer,ConvertSerializer,CompareSerializer,CheckLicenseSerializer,SubmitLicenseSerializer,ValidateSerializerReturn,ConvertSerializerReturn,CompareSerializerReturn,CheckLicenseSerializerReturn,SubmitLicenseSerializerReturn
 from api.oauth import generate_github_access_token,convert_to_auth_token,get_user_from_token
 from app.models import LicenseRequest
+from app.core import initialise_jpype, license_validate_helper, license_check_helper
```

Comment:
Import core instead of specific functions. Make it hard to maintain and also better understanding of code flow.